app/src/main/java/com/duckduckgo/app/pixels/AppPixelName.kt

```diff
@@ -244,6 +244,8 @@ enum class AppPixelName(override val pixelName: String) : Pixel.PixelName {
     EMAIL_DID_PRESS_WAITLIST_DIALOG_NOTIFY_ME("email_did_press_waitlist_dialog_notify_me"),
     EMAIL_DID_PRESS_WAITLIST_DIALOG_NO_THANKS("email_did_press_waitlist_dialog_dismiss"),
 
+    AUTOFILL_LOGINS_AUTOPROMPT_DISMISSED("m_autofill_logins_autoprompt_dismissed"),
```

Comment:
Would recommend moving this to inside `autofill-impl` module. Define a new enum which implements `Pixel.PixelName` and add your new pixel there